fuzzers/072-ordered_wires/run_fuzzer.py

```diff
@@ -0,0 +1,75 @@
+import threading
+import os
+import shutil
+import sys
+import subprocess
+import signal
+
+def start_vivado(blockID, start, stop):
+    print("Running instance :" + str(blockID))
+    try:
+        subprocess.check_call(
+            "${XRAY_VIVADO} -mode batch -source $FUZDIR/job.tcl -tclargs " +
+            str(blockID) + " " + str(start) + " " + str(stop),
+            shell=True)
+    except subprocess.CalledProcessError:
+        print("Problem happened in jobs !")
+        #dirty way to kill all the process
+        os.kill(os.getpid(), signal.SIGINT)
+
+def get_nb_pips():
+    print("Fetching nb pips")
+    try:
+        subprocess.check_call(
+            "${XRAY_VIVADO} -mode batch -source $FUZDIR/get_pipscount.tcl",
+            shell=True)
+    except subprocess.CalledProcessError:
+        print("Couldn't start vivado instance")
+        sys.exit(1)
+
```

Comment:
Don't swallow the exception.  Either don't catch the exception, or append to the exception